Code/GraphMol/Depictor/Templates.cpp

```diff
@@ -0,0 +1,90 @@
+//
+//  Copyright (C) 2023 Schrödinger, LLC
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+#include "Templates.h"
+
+namespace RDDepict {
+bool CoordinateTemplates::isValidTemplate(RDKit::ROMol& mol, const std::string& smiles) {
+    // template must have 2D coordinates
+    if (mol.getNumConformers() == 0) {
+        BOOST_LOG(rdWarningLog) << "WARNING: Template is missing coordinates: "
+                                << smiles << "\n";
+        return false;
+    }
```

Comment:
I kinda want to just throw instead of logging and bubbling up a boolean return in `setRingSystemTemplates`. If that's not the convention in this neighborhood of the RDK that's fine, but in general prefer `void setRingSystemTemplates` which throws on failure.

Reply:
I'm not opposed to that -- I think I started like this because I wanted to load some templates if someone had a file with only a couple invalid templates.

@greglandrum what do you think?

Reply:
I think it's perfectly fine to throw a ValueError here